internal/rbd/rbd_util_test.go

```diff
@@ -18,6 +18,9 @@ package rbd
 
 import (
 	"testing"
+	"strings"
```

Comment:
The linter is upset by this ordering, I think.

Reply:
Indeed, I would expect that too